examples/oneapi/cpp/source/pca/pca_cor_dense_batch.cpp

```diff
@@ -15,27 +15,22 @@
 *******************************************************************************/
 
 #include "oneapi/dal/algo/pca.hpp"
+#include "oneapi/dal/io/csv.hpp"
 
 #include "example_util/utils.hpp"
 
 using namespace oneapi;
 
-int main(int argc, char const *argv[]) {
-    constexpr std::int64_t row_count = 5;
-    constexpr std::int64_t column_count = 3;
+const char data_file_name[] = "../../daal/data/batch/pca_normalized.csv";
 
-    const float data[] = {
-        1.f,  2.f,  3.f,
-        1.f,  -1.f, 0.f,
-        4.f,  5.f,  6.f,
-        1.f,  2.f,  5.f,
-        -4.f, 3.f,  0.f
-    };
+int main(int argc, char const *argv[]) {
+    const auto data_source = dal::csv::data_source(data_file_name)
+        .set_delimiter(',');
```

Comment:
Can we avoid creating data source explicitly by the use of default params and leave only the call of the `read()` function?
Or this is done specifically for some reason?

Reply:
I believe there are now mechanisms during evaluating of read parameters to dispatch string into constructor of data_source, so it must be created explicitly. Moreover, data_source itself is a starting point for dispatcher to evaluate whole 'read' operation.

Reply:
Turned into one-liner.